api/rosetta/errors.go

```diff
@@ -42,11 +42,12 @@ const (
 	balancesRetrieval         = "unable to retrieve balances"
 	oldestRetrieval           = "unable to retrieve oldest block"
 	currentRetrieval          = "unable to retrieve current block"
+	intentDetermine           = "unable to determine transaction intent"
```

Comment:
This naming doesn't make sense to me, as shared in the other PR. Please notice how all of them are about a noun for an action (i.e. `retrieval`) combined with the thing the action is executed on. So it would probably be best to use `intentDetermination`.